cdc/kv/metrics.go

```diff
@@ -143,6 +143,21 @@ var (
 		},
 		// actions: wait, run.
 		[]string{"namespace", "changefeed"})
+
+	workerBusyRatio = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "ticdc",
+			Subsystem: "kvclient",
+			Name:      "worker_busy_ratio",
+			Help:      "Busy ratio (X ms in 1s) for region worker.",
```

Comment:
Please add more detail about what can we learn from this metric. It will be very helpful.